types/faker/faker-tests.ts

```diff
@@ -182,9 +240,19 @@ resultNum = faker.random.number({
     max: 0,
     precision: 0
 });
+resultNum = faker.random.float();
+resultNum = faker.random.float(0);
+resultNum = faker.random.float({
+	min: 0,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/faker/faker-tests.ts;
  Line: 246;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.